docs/workflow/debugging/libraries/unix-instructions.md

```diff
@@ -23,12 +23,7 @@ error: no such file
 
 It is also possible to debug .NET Core crash dumps using lldb and SOS. In order to do this, you need all of the following:
 
-- You will find the dump url to download the crash dump file in the test logs, something similar to:
-<pre>
-2017-10-10 21:17:48,020: INFO: proc(54): run_and_log_output: Output: dumplingid:  eefcb1cc36977ccf86f457ee28a33a7b4cc24e13
-2017-10-10 21:17:48,020: INFO: proc(54): run_and_log_output: Output: <b>https://dumpling.azurewebsites.net/api/dumplings/archived/eefcb1cc36977ccf86f457ee28a33a7b4cc24e13</b>
-</pre>
-- The crash dump file. We have a service called "Dumpling" which collects, uploads, and archives crash dump files during all of our CI jobs and official builds.
+- The crash dump file.
```

Comment:
Can we mention the new workflow to install the sos plugin or at least link to the new diagnostics repo? This section below still includes `"plugin load <path-to-libsosplugin.so>"` and other instances of libsosplugin.so like that one. 

https://github.com/dotnet/diagnostics/blob/master/documentation/installing-sos-instructions.md

cc: @mikem8361 

Now the sos plugin is loaded by default.

Reply:
Would you mind opening an issue to track this? These instructions are pretty stale anyway (references coreclr/corefx, etc) and I'm not an expert in what this should say. If you / @mikem8361 think just adding that link with a comment like "install SOS" is sufficient I'm happy to do that, but I don't think it's enough.

Reply:
You may what to refer to this doc: https://github.com/dotnet/diagnostics/blob/master/documentation/sos.md because it not only has the links to install SOS but points to info about lldb, etc.